package.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/ossf/package-feeds/blob/main/package.schema.json",
+    "title": "Package Schema Version 1.0",
+    "description": "The package representation as outputted by a ScheduledFeed",
+    "examples": [
+      "foobarpackage",
+      "1.0.0",
+      "2021-03-30T14:26:28",
+      "npm",
+      "1.0"
+    ],
+    "type": "object",
+    "properties": {
+      "name": {
+        "type": "string",
+        "description": "The name of the package"
+      },
+      "version": {
+        "type": "string",
+        "description": "The package version, formatted respective to the given package type"
+      },
```

Comment:
On one hand, each package maintainer is free to use whichever versioning schema they choose fit, so free-text should definitely be accepted. But package versions complying with [semantic versioning](https://semver.org/) makes downstream compute/lookup much easier. As such, it would be awesome to add a flag mentioning that the version string can be parsed with semver libraries.

Reply:
I would prefer to avoid having a flag to indicate semver compatible version strings as this can be checked on the consumer side just as easily. Unfortunately semver usage is down to the maintainers of individual packages as you've mentioned, so we can't even define this on a per-feed basis.